modules/cnv-about-kubevirtstorageclassdefaults.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_virtual_machines/cnv_virtual_disks/cnv-storage-defaults-for-datavolumes.adoc
+
+[id="cnv-about-kubevirtstorageclassdefaults_{context}"]
+= About storage settings for DataVolumes
+
+DataVolumes require a defined access mode and volume mode to be created in the web console. 
```

Comment:
These settings are only required to create DVs in the web console, but defaults are configured at installation so users don't need to change them. The only way I can see to avoid this passivity is through unnecessary wordiness. I think this is the lesser of two evils as it's more straightforward and concise.